lms/djangoapps/django_comment_client/utils.py

```diff
@@ -62,6 +63,15 @@ def has_forum_access(uname, course_id, rolename):
     return role.users.filter(username=uname).exists()
 
 
+def has_required_keys(module):
+    """Returns True iff module has the proper attributes for generating metadata with get_discussion_id_map_entry()"""
+    for key in ('discussion_id', 'discussion_category', 'discussion_target'):
+        if getattr(module, key, None) is None:
+            log.warning("Required key '%s' not in discussion %s, leaving out of category map", key, module.location)
```

Comment:
this should log at `DEBUG` unless it's something that developers / ops should be concerned with every time it happens.